app/models/game.rb

```diff
@@ -4,4 +4,89 @@ class Game < ApplicationRecord
   belongs_to :player_1, class_name: 'Player', optional: true
   belongs_to :player_2, class_name: 'Player', optional: true
   belongs_to :winning_player, class_name: 'Player', optional: true
+  # Is called after  Base.save on new objects that have not been saved yet (no record exists).
+  after_create :populate_game!
+  # How to fix messed up migrations http://stackoverflow.com/questions/20082002/migrations-are-pending-run-bin-rake-dbmigrate-rails-env-development-to-resol
+  # Populate a game with all the pieces in the correct locations (x_coordinate, y_coordinate)
+
+  def populate_game!
+    add_starting_pieces_for_color('white')
+    add_starting_pieces_for_color('black')
+  end
+
+  def add_starting_pieces_for_color(_color)!
```

Comment:
Do you mean the bang(!) or the underscore?  The underscore is put in by rubocop, I just noticed that.